src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerHwfPartPaymentRequired.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.divorce.caseworker.event;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.sdk.api.CCDConfig;
+import uk.gov.hmcts.ccd.sdk.api.ConfigBuilder;
+import uk.gov.hmcts.divorce.common.ccd.PageBuilder;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.divorcecase.model.State;
+import uk.gov.hmcts.divorce.divorcecase.model.UserRole;
+
+import static uk.gov.hmcts.divorce.divorcecase.model.State.AwaitingHWFDecision;
+import static uk.gov.hmcts.divorce.divorcecase.model.State.AwaitingHWFPartPayment;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.CASE_WORKER;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.LEGAL_ADVISOR;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.SUPER_USER;
+import static uk.gov.hmcts.divorce.divorcecase.model.access.Permissions.CREATE_READ_UPDATE;
+
+@Component
+public class CaseworkerHwfPartPaymentRequired implements CCDConfig<CaseData, State, UserRole> {
+
+    public static final String CASEWORKER_HWF_Part_Payment_Required = "caseworker-hwf-part-payment-required";
```

Comment:
Can this variable name be all UPPER case

Reply:
Thanks for the comment @amoriartyMOJ i'll update the variable.